cloud-functions-http/container/translations/shell-translations.csv

```diff
@@ -53,41 +53,41 @@ Cloud 함수의 이름은 실행하려는 메서드의 이름과 동일해야 
  注意：部署功能可能需時幾分鐘。"
 DEPLOYMENT_HINT,"The function you want to deploy is called $FUNCTION
 
-You want to set the function to triger on http requests, which means you want to use the --trigger-http flag
+You want to set the function to trigger on http requests, which means you want to use the --trigger-http flag
```

Comment:
This file is not the canonical source for these translations. The canonical source is kept in a Google Sheet. I've made the updates there, so we should be good to go.

Reply:
Good to know! I will set-up the new tracks with translations in a Google Sheet too.